scripts/in_nic/vocab/pull_data.py

```diff
@@ -0,0 +1,60 @@
+import requests
+import json
+import pandas as pd
+
+url =  "http://vocab.nic.in/rest.php/"
+
+data_files = {
+    'States': 'states/',
+    'Country':  'country/',
+    'Districts': 'district/tn/',
+    'Pincode Details': 'pincode/110003/',
+    'Taluks': 'taluk/tn/',
+    'Language': 'lang/',
+    'Taluks (District - wise)': 'taluk/tn/district/TN003/',
+    'Audiences': 'audience/',
+    'Sectors': 'sectors/',
+    'Sub Sectors': 'sectors/subsectors/',
+    'Site Count': 'sitecount/',
+    'Sub Sectors (Sector-wise)': 'sectors/ST01/subsectors/'
+}
+
+
+class VocabularyDataLoader:
+    def __init__(self, data_files, url):
+        """
```

Comment:
Starts on the same line.

For example:
`"""xxxxxxxxxxxxxxxxxxxx
xxxxxxxxxx.

xxxxxxxxxxxxxxxx.
"""`

Reply:
and Args and Returns

Reply:
http://google.github.io/styleguide/pyguide.html#383-functions-and-methods